ls1intum/Artemis
# Feature/search user in usermanagement

### Checklist

### Motivation and Context
In den User Management Overview, an admin gets all users of Artemis listed. On the Testserver we have around 2500 users. This user number requires a search field to find a specific user in case you need to make any changes to it. The search field considers the email and login. Filtering is made at the server. 

Solves #1220 

### Description
Login to Artemis as Admin. Navigate to "User management" under the "Service Administration" drop down. Insert a search string in the input field. The Result needs to contain the search string either in the login and/or in the email. 

### Steps for Testing
1. Log in to Artemis as Admin
2. Under Service Administration navigate to User management 
3. Type in a search string in  the input field (e.g. krusche)
4. The results should only contain elements containing the search string.
5. Test if filtering over different columns works as expected

### Screenshots
![FirstScreenshot](https://user-images.githubusercontent.com/23211992/77094302-94fd4700-6a0c-11ea-8577-d131d9715e94.png)

![Screenshot2](https://user-images.githubusercontent.com/23211992/77097126-96c90980-6a10-11ea-8c8f-1bf7f6fd3cb0.png)
